src/main/java/BMSStreamSender/BMSSender.java

```diff
@@ -0,0 +1,28 @@
+package BMSStreamSender;
+
+public class BMSSender {
+
+	public static void main(String[] args) {
+		
+		System.out.println("Please enter number of BMS readings to be generate (Min->10):");
```

Comment:
The sender's output goes to the Receiver due to the pipe. So in production, this println will not be seen at all?